.github/workflows/Trgr InitDev workflows in Implementation repo.yml

```diff
@@ -57,8 +57,29 @@ jobs:
           $current_branch = '${{env.HEAD_REF}}'
         }
         Write-Host "Current Branch: $current_branch"   
-        echo "current_branch=$current_branch" >> $Env:GITHUB_ENV                   
+        echo "current_branch=$current_branch" >> $Env:GITHUB_ENV
+    - name: Check Ed-Fi-ODS-Implementation Repo PR exists or not
+      shell: pwsh
+      run: |
+        $accessToken = "${{ env.EDFI_ODS_IMP_TOKEN  }}"
+        $apiUrl = "https://api.github.com/repos/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/pulls?state=open&head=Ed-Fi-Alliance-OSS:${{env.current_branch}}"
+        $headers = @{
+            Authorization = "Bearer $accessToken"
+            Accept = "application/vnd.github.v3+json"
+        }
+        $prList = Invoke-RestMethod -Uri $apiUrl -Headers $headers
```

Comment:
Can you also capture and print the rate limit headers, which might be useful for diagnostics? See https://docs.github.com/en/rest/overview/rate-limits-for-the-rest-api?apiVersion=2022-11-28#checking-the-status-of-your-rate-limit